packages/firebase_ml_vision/test/firebase_ml_vision_test.dart

```diff
@@ -625,175 +621,6 @@ void main() {
       });
     });
 
-    group('$LabelDetector', () {
```

Comment:
The code below was moved to its own file `image_labeler_test.dart` changes can be seen [here](https://github.com/flutter/plugins/pull/1364/commits/c9c42d8175dd30f21243d926165af466e887f590).